ResultadosDigitais/active_merchant
# Moving worldpay to the latest version

Adding changes to Worldpay driver to turn him more closer to the current version of ActiveMerchant master branch.

Later we will open a PR to ActiveMerchant with these modifications.

To test this files:
```
rvm install 2.5.4
bundle install --jobs=4 --retry=3 --gemfile Gemfile.rails42 --path=vendor/bundle_42

BUNDLE_GEMFILE=Gemfile.rails42
bundle check --path=vendor/bundle_42 
bundle exec rake test #3 test of another gateways that we don't use may be broken, probably due our outdated version of rd_improvements branch
```

WARNING: Please, do not merge until we merge PR [5045](https://github.com/ResultadosDigitais/saas-internal/pull/5045) from Saas-Internal
